uber/templates/preregistration/preregbase.html

```diff
@@ -64,12 +66,12 @@
             row: '#badge-types',
             selector: '.badge-type-selector',
             options: [{
```

Comment:
Brent pointed out that on the confirmation page people might worry they weren't staffers if it just said "Attending" at the top.  So now for staff badges it says "Staffer" and "Attending" otherwise, which seems like a sensible default.